### ISSUE TITLE
npm install eslint when docker build is useless

### ISSUE THREAD

**Issue by LaurenceLiZhixin** [eslint/eslint#13125]
**Tell us about your environment**

* **ESLint Version:6.8.0**
* **Node Version:v12.16.1**
* **npm Version:6.13.4**

**What parser (default, Babel-ESLint, etc.) are you using?**

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**
I just want to build my docker using dockerfile below

```bash
FROM dockerhub.azk8s.cn/library/node:12.16.1
RUN apt-get update
RUN apt-get install git
RUN npm i -g eslint eslint-plugin-react@latest @typescript-eslint/parser @typescript-eslint/eslint-plugin --registry=https://registry.npm.taobao.org
```
when I build my docker. All went right. 
But when I use my docker, I run 'eslint .'
It shows that:
```
Oops! Something went wrong! :(
ESLint: 6.8.0.
ESLint couldn't find the plugin "eslint-plugin-react".
(The package "eslint-plugin-react" was not found when loaded as a Node module from the directory "/builds/brv/brv-investment-ui".)
It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:
    npm install eslint-plugin-react@latest --save-dev
The plugin "eslint-plugin-react" was referenced from the config file in ".eslintrc.js".
If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.
ERROR: Job failed: command terminated with exit code 1
```
Only when running my docker as container can I install the plugin above, and only in this way the eslint runs normally. 
It costs plenty of time when install the plugin in run time. I just want to build the plugin in my docker and run freely.
But -g flag is useless, the plug doesn't install.

**Comment by kaicataldo** [comment]
Plugins are resolved relative to the configuration file, so we recommend that ESLint be installed and executed locally per-project using either `./node_modules/.bin/eslint .` or `npx eslint .`

**Comment by nzakas** [comment]
It looks like the conversation is stalled here. As this is a question rather than an action item, I'm closing the issue. If you still need help, please send a message to our [mailing list](https://groups.google.com/group/eslint) or [chatroom](https://gitter.im/eslint/eslint). Thanks!
